packages/eslint-plugin-ckeditor5-rules/lib/rules/validate-changelog-entry.js

```diff
@@ -125,6 +125,26 @@ module.exports = {
 							}
 						} );
 					} );
+
+				doc.errors
```

Comment:
If we're adding a validator, I think it would be better to validate the indentation the right way. Right now it is possible to use any amount of spaces:

<img width="943" height="452" alt="Image" src="https://github.com/user-attachments/assets/949975a2-a4d7-425c-9c00-246c1919452e" />

Maybe let's use some libraries that Kamil suggested in the ticket's comments.

Reply:
Well, we can use a plugin, but that would require making PRs in all repositories to add the plugin to their respective eslint configs, as the changeset validator is not enabled in our global ckeditor ruleset.

> Right now it is possible to use any amount of spaces:

Isn't that out of scope of this ticket? I believe it only specified to check that there is no tabs

Reply:
> Well, we can use a plugin, but that would require making PRs in all repositories to add the plugin to their respective eslint configs, as the changeset validator is not enabled in our global ckeditor ruleset.

So maybe it's time to enable this rule globally, since we are using Yaml-based changeset format globally.

Reply:
I think we should avoid implementing our own version of indentation checks if there is already open-source and popular validator.